CHANGELOG.md

```diff
@@ -44,7 +44,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Fixed
 
-- Nothing.
+- fixes case sensitivity for SameSite directive.
```

Comment:
Can be fixed on merge: it should contain link to the PR as we have for other changelogs